<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>member_edit</class>
 <widget class="QDialog" name="member_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>회원정보 수정</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>101</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>아이디</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>비밀번호</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>닉네임</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>포인트</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>쿠폰</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>211</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="id_txt">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pw_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="name_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="point_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="coupon_txt"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>99</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>등급</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="common_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>일반</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="premium_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>premium</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vip_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>vip</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
